aiplatform/src/main/java/aiplatform/CreateCustomJobSample.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aiplatform;
+
+// [START aiplatform_create_custom_job_sample]
+import com.google.cloud.aiplatform.v1.AcceleratorType;
+import com.google.cloud.aiplatform.v1.ContainerSpec;
+import com.google.cloud.aiplatform.v1.CustomJob;
+import com.google.cloud.aiplatform.v1.CustomJobSpec;
+import com.google.cloud.aiplatform.v1.JobServiceClient;
+import com.google.cloud.aiplatform.v1.JobServiceSettings;
+import com.google.cloud.aiplatform.v1.LocationName;
+import com.google.cloud.aiplatform.v1.MachineSpec;
+import com.google.cloud.aiplatform.v1.WorkerPoolSpec;
+import java.io.IOException;
+
+public class CreateCustomJobSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "PROJECT";
+    String displayName = "DISPLAY_NAME";
+    String containerImageUri = "CONTAINER_IMAGE_URI";
+    createCustomJobSample(project, displayName, containerImageUri);
+  }
+
+  static void createCustomJobSample(String project, String displayName, String containerImageUri)
+      throws IOException {
+    JobServiceSettings settings =
+        JobServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+    String location = "us-central1";
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
```

Comment:
Since we are using Try with resources here; we do not need the statement about `close`. Once the try block is finished the close will be called automatically.  So we can remove the last line of comment here

Reply:
Changed